fiftyone/utils/labelbox.py

```diff
@@ -2121,7 +2121,9 @@ def _make_mask(instance_uri, color):
 # https://labelbox.com/docs/exporting-data/export-format-detail#video
 def _parse_video_labels(video_label_d, frame_size):
     url_or_filepath = video_label_d["frames"]
-    label_d_list = _download_or_load_ndjson(url_or_filepath)
+    headers = {"Authorization": "Bearer %s" % API_KEY}
```

Comment:
`API_KEY` isn't defined here.

I guess one approach would be to allow it to be manually provided as a kwarg. However, our [Labelbox integration](https://voxel51.com/docs/fiftyone/integrations/labelbox.html#authentication) provides a way to access one's API key as follows:

```shell
export FIFTYONE_LABELBOX_API_KEY=XXXXXXXX
```

```py
import fiftyone as fo

print(fo.annotation_config.backends["labelbox"]["api_key"])
# XXXXXXXX
```

Reply:
Yes, I wasn't sure about if it is ok to provide api_key as a kwarg, but to me the second solution (exporting api key) makes more sense (then we will need to add that as an instruction for calling the 'import_from_labelbox' method).